crates/starknet-server/src/api/json_rpc/endpoints.rs

```diff
@@ -0,0 +1,139 @@
+use crate::api::models::contract_class::ContractClass;
+use crate::api::models::transaction::{
+    BroadcastedTransactionWithType, ClassHashHex, EventFilter, EventsChunk, FunctionCall,
+    Transaction, TransactionHashHex, TransactionReceipt, TransactionWithType,
+};
+use crate::api::models::{block::Block, state::ThinStateDiff, BlockId};
+use crate::api::models::{ContractAddressHex, FeltHex, PatriciaKeyHex};
+
+use starknet_types::starknet_api::block::BlockNumber;
+
+use super::models::{BlockHashAndNumberOutput, EstimateFeeOutput, SyncingOutput};
+use super::RpcResult;
+
+use super::{
+    error::{self},
+    JsonRpcHandler,
+};
+
+/// here is definiton and stub implementation of all JSON-RPC endpoints
+impl JsonRpcHandler {
+    pub(crate) async fn get_block_with_tx_hashes(&self, _block_id: BlockId) -> RpcResult<Block> {
```

Comment:
Wdyt we add above all the endpoints the name of the RPC endpoint from the specs as a comment, like:
starknet_getBlockWithTxHashes

Reply:
Seems redundant if we will always just be applying the naming formula of stripping `starknet_` and converting camel- to snake-case.

Reply:
sure, but if you go and search the spec, and want to see that same endpoint here, you cannot just copy paste, but have to modify to our format

Reply:
Added in [262aa19](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/8/commits/262aa19d1294aa0d1ba4e2d72c88a3e538fbfecd)